liquibase/changeLogs/capture/tables/fieldVisitHeaderInfo/fieldVisitHeaderInfo.sql

```diff
@@ -0,0 +1,16 @@
+create table if not exists ${AQTS_SCHEMA_NAME}.field_visit_header_info
+(field_visit_header_info_id                bigint generated by default as identity (start with 1)
+	,json_data_id                            bigint
+	,field_visit_identifier                  text
```

Comment:
This is `identifier` in the header info section of the sv json.  It seems like an aqts auto-generated id and is unclear that we have any use for it.